docs/get-started/tooling/attestations/primus.mdx

```diff
@@ -0,0 +1,61 @@
+title: PRIMUS
+image: /img/socialCards/primus.jpg
+---
+
+[PRIMUS](https://www.primuslabs.xyz) is a cryptography-based attestation protocol to bring all Internet
+data into smart contracts. PRIMUS applies advanced cryptography to minimize trust and enables end
+users to prove the correctness of their personal web data in a privacy-preserving manner.
+
+The core techniques used in PRIMUS include zkTLS and zkFHE. End users run PRIMUS client apps to attest their personal web data and create related data proofs with the help of PRIMUS nodes. These data proofs are cryptographically verified attestations with security guarantees including:
+
+- **Data authenticity**: The user web data is retrieved from the indicated data source with verified integrity
+secured by zkTLS and other advanced cryptographic techniques.
+- **Privacy**: When coupled with specific application logic users' sensitive information is
+well-protected under the zero-knowledge property.
+
+The zkTLS and zkFHE techniques are general enough for both data source integration and data
+computation. You can create your own attestations such as:
+
+- Proofs of CEX assets: Alice has more than 1000 USD for her total token values in Coinbase
+- Proofs of specific token holding: Alice holds more than 2 ETH in Binance
+- Proofs of nationality: Alice is a verified bank user with British nationality
+- Proofs of gaming activity: Bob plays more than 10 hours per day on Steam games
+
+## For developers
+
+The PRIMUS team is creating a template-based SDK (in progress) for developers to build their own
+applications using PRIMUS' off-chain attestation capabilities. The SDK will encapsulate the details
+of underlying cryptographic algorithms and protocols and provide high-level interfaces for
+application developers to manage the data retrieval and data computation. Developers can leverage
+the SDK in two aspects:
+
+- Use the data source templates to integrate with any preferred Internet data source and to attest
+the corresponding type of data.
+- Define and implement their own application logic by integrating with data consumption templates.
+
+## Networks
+
+PRIMUS is deployed on Linea and integrated with [Verax](https://github.com/Consensys/verax-documentation)
+a public attestation registry to store and manage attestation data that can be accessed by any
+party. This means PRIMUS users can submit these data proofs to Linea via Verax which can be further
+accessed by other dapps. The following contract address is the PRIMUS portal contract:
+
+| Network | Contract address |
+| --- | --- |
+| Linea Mainnet | [`0x50bd377EB8D4236Bb587AB3FB1eeafd888AEeC58`](https://lineascan.build/address/0x50bd377EB8D4236Bb587AB3FB1eeafd888AEeC58) |
+
+## Resources
+
+Learn more and start using PRIMUS!
+
+- [Product tutorial](https://docs.primuslabs.xyz/)
+- [Technical paper](https://eprint.iacr.org/2023/964.pdf)
+
+## Learn more
+
+- [Twitter](https://x.com/primus_labs)
+- [Discord](https://discord.gg/YxJftNRxhh)
```

Comment:
I think we need to remove this — the link is invalid and I can't see a Discord link on their site

Reply:
@qvkare does that link work for you? still invalid for me